packages/run-protocol/src/runStake/runStake.js

```diff
@@ -0,0 +1,171 @@
+// @ts-check
+import { handleParamGovernance } from '@agoric/governance';
+import { E, Far } from '@endo/far';
+import { makeAttestationFacets } from './attestation.js';
+import { makeRunStakeParamManager } from './params.js';
+import { makeRunStakeKit } from './runStakeKit.js';
+import { makeRunStakeManager } from './runStakeManager';
+
+/**
+ * Provide loans on the basis of staked assets that earn rewards.
+ *
+ * In addition to brands and issuers for `Staked`, `RUN`, and attestation,
+ * terms of the contract include a periodic interest rate plus
+ * a fee proportional to the amount borrowed, as well as a ratio
+ * of funds to mint and loan per unit of staked asset.
+ * These terms are subject to change by the `Electorate`
+ * and `electionManager` terms.
+ *
+ * @typedef {{
+ *   electionManager: VoteOnParamChange,
+ *   main: {
+ *     MintingRatio: ParamRecord<'ratio'>,
+ *     InterestRate: ParamRecord<'ratio'>,
+ *     LoanFee: ParamRecord<'ratio'>,
+ *     Electorate: ParamRecord<'invitation'>,
+ *   },
+ * }} RunStakeGovernanceTerms
```

Comment:
Since `Electorate` and `electionManager` are part of all governance so for documentation purposes here it may be better to separate out,
```
@typedef {{
     MintingRatio: ParamRecord<'ratio'>,
     InterestRate: ParamRecord<'ratio'>,
     LoanFee: ParamRecord<'ratio'>,
}} RunStakeParamTerms
```
then,
```
 * @typedef {{
 *   electionManager: VoteOnParamChange,
 *   main: RunStakeParamTerms & {
 *     Electorate: ParamRecord<'invitation'>,
 *   },
 * }} RunStakeGovernanceTerms
```
The simplification PR has a utility type for the latter.